zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -47,6 +47,7 @@
         }
       }
       $scope.defaultVersions = defaultVersions;
+      console.log(_.size(defaultVersions));
```

Comment:
I thought we generally avoid console.log unless absolutely necessary (like error)